src/main/java/frc/robot/subsystems/CrocodileSubsystem.java

```diff
@@ -1,92 +1,68 @@
 package frc.robot.subsystems;
 
+import javax.swing.text.Position;
+
 import com.ctre.phoenix.motorcontrol.NeutralMode;
-import com.ctre.phoenix.motorcontrol.StatorCurrentLimitConfiguration;
 import com.ctre.phoenix.motorcontrol.SupplyCurrentLimitConfiguration;
 import com.ctre.phoenix.motorcontrol.can.WPI_TalonFX;
-import edu.wpi.first.math.filter.Debouncer;
-import edu.wpi.first.wpilibj.Solenoid;
+
+import edu.wpi.first.math.controller.PIDController;
+import edu.wpi.first.wpilibj.DigitalInput;
+import edu.wpi.first.wpilibj.DutyCycleEncoder;
+import edu.wpi.first.wpilibj2.command.PIDSubsystem;
 import edu.wpi.first.wpilibj2.command.SubsystemBase;
-import frc.robot.Constants;
 import frc.robot.RobotMap;
 
 public class CrocodileSubsystem extends SubsystemBase {
-    private final Solenoid wristSolenoid;
-    private final Solenoid chomperSolenoid;
     private final WPI_TalonFX intake;
-    private Debouncer stallDebouncer = new Debouncer(0.050, Debouncer.DebounceType.kRising);
-    private double currentMotorPower;
+    private final WPI_TalonFX wrist;
+    private final PIDController pid;
+    private final DutyCycleEncoder encoder;
+    private final DigitalInput beamBreak;
+
+    //TODO: Set speed limit
+    private final double SPEED_LIMIT = 0;
 
     public CrocodileSubsystem() {
         intake = new WPI_TalonFX(RobotMap.Crocodile.INTAKE);
+        wrist = new WPI_TalonFX(RobotMap.Crocodile.WRIST);
+        pid = new PIDController(0, 0, 0);
```

Comment:
I would name the controller to something a bit more descriptive. What does it controller?